pages/data-design/avo-tracking-plan/journeys.mdx

```diff
@@ -65,7 +71,9 @@ For a full walkthrough of building journeys on the canvas, see [Creating a journ
 
 ### Journey triggers
 
-Journey triggers represent the navigation, system, or user action on a journey step that triggers the event to be sent. One journey step can be connected to one or more triggers and one trigger can have one or more journey steps connected to it. Triggers are connected to events and/or variants in your tracking plan. 
+Journey triggers represent the navigation, system, or user action on a journey step that triggers the event to be sent. One journey step can be connected to one or more triggers and one trigger can have one or more journey steps connected to it. Triggers are connected to events in your tracking plan. 
+
+![Journey triggers](/images/journeys/property-conditions.png)
```

Comment:
Nit: Have one of the trigger nodes focused instead of the step node

Reply:
not going to recreate now